Rakefile

```diff
@@ -44,7 +44,6 @@ end
 # restart
 %w[
   app:template
-  app:update
```

Comment:
Rails 7.2 updates this Rake Task to now be a Rails command. Causing this line to error out as it is not a defined task after Rails 7.2. This command could be added to the setup file, but as we only need to run this when updating the rails version i don't see a need for this in the automated setup.